### PR TITLE
[WiP] revamp docs 

### PR DESCRIPTION
**Pull request by nav-github01001**
## Description

Docs are completely not working, reinstalling them

## Checklist

- [ ] - All code changes have been tested, if any
- [ ] - The documentation has been updated accordingly for any code changes
- [ ] - This PR fixes an already existing issue
- [ ] - This PR adds a new feature (new functions, methods, classes, parameters, etc.)
- [ ] - This PR fixes a bug
- [ ] - This PR is a breaking change (removes/renames functions, parameters etc.)
- [ ] - This PR is NOT a code change (changes Documentation files, README.md, CONTRIBUTING.md etc.)

### CODE REVIEW

File: EpikCord/__main__.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
 __all__ = (
     
 )
+import re
 
-def __version__() -> str:
-    return "0.4.13.3"
+
+__version__= '0.4.11'
```
Comment (TheUntraceable) [inline EpikCord/__main__.py:7]: This is 4.13.3

Comment (nav-github01001) [reply EpikCord/__main__.py:7]: oops

**Review by TheUntraceable — APPROVED** [review]

**Comment by nav-github01001** [comment]
@TheUntraceable commit it

**Comment by TheUntraceable** [comment]
Add back the EpikCord image

**Comment by nav-github01001** [comment]
O shit

**Comment by nav-github01001** [comment]
4got that

**Review by TheUntraceable — APPROVED** [review]

**Pull request merged**